src/uiSelectDirective.js

```diff
@@ -171,6 +171,12 @@ uis.directive('uiSelect',
           $select.spinnerClass = spinnerClass !== undefined ? attrs.spinnerClass : uiSelectConfig.spinnerClass;
         });
 
+        attrs.$observe('appendDropdownToBody', function() {
+          // $eval() is needed otherwise we get a string instead of a boolean
+          var appendDropdownToBody = scope.$eval(attrs.appendDropdownToBody);
+          $select.appendDropdownToBody = appendDropdownToBody !== undefined ? appendDropdownToBody : uiSelectConfig.appendDropdownToBody;
+        });
```

Comment:
do you need to worry about this changing while something is UP and appendedToBody and  now you need to reparent it? my guess is no.